Implementations/Algorithms/Pong_REINFORCE.py

```diff
@@ -140,7 +158,7 @@ def update_policy(self, rewards, log_probs, last_g, last_b, last_log_probs):
         discounted_rewards = (discounted_rewards - discounted_rewards.mean()) / (discounted_rewards.std() + 1e-9) # normalize discounted rewards
 
         policy_gradient = []
-        for log_prob, Gt in zip_longest(log_probs, discounted_rewards):
+        for log_prob, Gt in zip(log_probs, discounted_rewards):
```

Comment:
When not using reward-to-go, the above change ends up truncating to the shortest of the two lists. That's not a desired outcome for this case, as we want `policy_gradient` to compute the sum of all `log_probs`.

Reply:
done